backend/index.ts

```diff
@@ -72,15 +72,35 @@ const _controllers = [
   new WebhookController(privateApp),
 ];
 
-ContentWatcherService.getInstance().then((service) => {
+async function waitForContentWatcherService(service: ContentWatcherService, maxWaitTime: number): Promise<void> {
+  let isReady = false;
+  for (let i = 0; i < maxWaitTime; i++) {
+    isReady = await service.getScannerReadyStatus();
+    if (isReady) {
+      break;
+    }
+    await new Promise((resolve) => setTimeout(resolve, 1000));
+  }
+
+  if (!isReady) {
+    logger.error(`Content Watcher Service not ready after ${maxWaitTime} seconds. Exiting app.`);
+    throw new Error('Content Watcher Service not ready');
+  }
```

Comment:
I wonder if the following makes sense, instead of waiting for a fixed amount of time:
* Have the function loop forever waiting for the content-watcher, instead of throwing an error after a time. Make the readiness probe `/readyz` dependent on this successfully completing

Reasoning:
* Availability of the content-watcher is not an internal application error, and likely will not be fixed by the service being restarted
* Why `/readyz` instead of `/livez`? Same as above--`/livez` failure would cause K8s to restart the pod; this likely won't help the situation. But `/readyz` failure should prevent us from routing traffic to the pod until it is fully up.

Reply:
Noted in #100